src/components/heading/heading.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { shallow, mount } from "enzyme";
+import { shallow, mount, ReactWrapper } from "enzyme";
 import Heading from "./heading.component";
```

Comment:
you could change that to import the component from the folder, that will ensure we are exporting it properly in the index file, when someone do modify it in the future

```suggestion
import Heading from ".";
```